akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -111,6 +118,62 @@ class AskSpec extends ScalaTestWithActorTestKit("""
       }
     }
 
+    "fail the future if the actor has terminated" in {
+      val actor: ActorRef[Msg] = spawn(behavior)
+
+      val deadLetterProbe = createDeadLetterProbe()
+      val probe = createTestProbe[Any]()
+      actor ! Stop(probe.ref)
+      implicit val timeout: Timeout = 10.millis
+
+      val answer: Future[String] = actor.ask(Foo("bar", _))
```

Comment:
Simulate ask a terminated actor, it should publish a DeadLetter with the recipient.